iot/google/cloud/iot_v1/gapic/device_manager_client.py

```diff
@@ -1491,7 +1495,7 @@ def get_iam_policy(
     def test_iam_permissions(
         self,
         resource,
-        permissions,
+        permissions=None,
```

Comment:
I don't understand this change:  the only point of `test_iam_permissions` is to pass through a set of permissions, and get back the ones which are allowed:  making it optional is just silly.